schemas/ifdq_ifcdaq_data.fbs

```diff
@@ -0,0 +1,16 @@
+//This schema is under development.
+//If you decide to use it, be aware that backwards compatibility is not guaranteed.
+
+file_identifier "ifdq";
+
+namespace FSD;
+
+table ifcdaq_data {
+    pv: string (required);
+    timestamp: ulong;
+    value: [double] (required);
+    time: [double] (required);
+    uniqueId: int;
```

Comment:
What is the use case for `uniqueId`?
I'm just curious because we decided against having something like that in the order schema.

Reply:
I am not a 100% sure but I think that it is a frame counter for making sure that no data is lost. Regardless, I think it does not matter very much at the moment seeing as the schema is under development. :-)